src/oop/object_independence/medium/q4/QuestionText.md

```diff
@@ -0,0 +1,33 @@
+# oop-object_independence-medium-q4
+
+## Question Info
+
+- Question type: Java
+
+### Question Text
+
+You are about to take a long road trip so you want to build a simple portable player to keep yourself entertained on the 
+trip. Make a class called **portableMusicPlayer** with the following description:
+
+### PortableMusicPlayer Class
+
+Your `PortableMusicPlayer` class must have the following items:
+
+- Attributes:
+    - A non-empty array of `String` called **playlist**
```

Comment:
I don't think saying non-empty is helpful here. I think on the methods just say to assume that the **playlist** is non-empty, or add something like "No music playing" if there is none.